src/stylesheets/layouts/_concept.scss

```diff
@@ -0,0 +1,380 @@
+.solution {
+  background: linear-gradient($color-medium-blue 5%, $color-mint 90%);
+  @include flex;
+
+  &__wrapper {
+    position: relative;
+    @include flex;
+    width: 100%;
+    height: 100%;
+    max-width: 77rem;
+    margin: 0;
+    padding: 2.5rem 1.5rem;
+    @media (min-width: 550px) {
+      margin: 2rem;
+      padding: 4rem 6.5rem;
+    }
+    @media (min-width: 800px) {
+      margin: 2rem 2rem 0;
+      padding: 5rem 8.5rem 0;
+    }
+  }
+
+  &__florence {
+    width: 200px;
+    margin-top: -30%;
+    margin-bottom: 30px;
+  }
+
+  &__hdr {
+    margin-bottom: $content-margin-btm;
+
+    &--top {
+      display: block;
+      padding-bottom: $hdr-padding-btm;
+    }
+
+    &--btm {}
+  }
+
+  &__context {
+    margin-bottom: $content-margin-btm;
+  }
+
+  &__section {
+    &--1,
+    &--2,
+    &--3,
+    &--4,
+    &--5,
+    &--6,
+    &--7 {
+      position: relative;
+      display: flex;
+      flex-direction: column;
+      align-items: center;
+      width: 100%;
+      background-color: $color-white;
+      margin: 7rem 0;
+      padding: 5rem 1.5rem;
+      @media (min-width: 800px) {
+        flex-direction: row;
+        margin: 5rem;
+        max-width: 77rem;
+        background-color: $color-white;
+        min-height: 39.5rem;
+        padding: 5rem 8.5rem;
+        border-radius: 5px;
+      }
+    }
```

Comment:
Usually with BEM, this element would have both a `solution__section` class _and_ a `solution__section--1` class.

Apply your shared CSS to `solution__section` and only apply unique CSS to the `solution__section--#` classes

Reply:
resolved in pr#4